package.json

```
@@ -1872,7 +1872,7 @@
             ],
             "aws.auth": [
                 {
-                    "command": "aws.auth.addConnection",
+                    "command": "aws.auth.showConnectionsPage",
```

Similar to the "form" , no need to engrave the current delivery mechanism in the command name.

```suggestion
                    "command": "aws.auth.manageConnections",
```